src/agent/agent_info/src/agent_info_persistance.cpp

```diff
@@ -75,6 +86,22 @@ void AgentInfoPersistance::CreateAgentInfoTable()
     }
 }
 
+void AgentInfoPersistance::CreateAgentGroupTable()
```

Comment:
Do we need a table for the agent group? Can't it be just another column in the `agent_info` table?

Reply:
I think that in the future more information related to the groups could be added, for example the hash of the configuration file of each group to check that it has not been modified. That's why I thought about adding a new table.